src/main/scala/scalismo/common/Vectorizer.scala

```diff
@@ -0,0 +1,15 @@
+package scalismo.common
+
+import breeze.linalg.DenseVector
+
+/**
+ * Created by forand00 on 03.05.16.
+ */
+trait Vectorizer[Value] {
+  def dim: Int
```

Comment:
i think length or size would suggest that it is related to the vertorizer itself. But instead it refers to the dimension (size/length) of the dense vector representing the value type. Therefore i would argue for to leave it as dim.